app/models/playlist_item.rb

```diff
@@ -1,6 +1,11 @@
 class PlaylistItem < ActiveRecord::Base
   belongs_to :stream
   validates :song, uniqueness: true, unless: :last_song_differs
+  after_create {|item| item.notify }
```

Comment:
Space between { and | missing.<br>Pass `&:notify` as an argument to `after_create` instead of a block.